plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/GetMobileApp.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Automattic\WooCommerce\Admin\Features\OnboardingTasks\Tasks;
+
+use Automattic\WooCommerce\Admin\Features\OnboardingTasks\Task;
+
+/**
+ * Get Mobile App Task
+ */
+class GetMobileApp extends Task {
+	/**
+	 * ID.
+	 *
+	 * @return string
+	 */
+	public function get_id() {
+		return 'get-mobile-app';
+	}
+
+	/**
+	 * Title.
+	 *
+	 * @return string
+	 */
+	public function get_title() {
+		return __( 'Get the free WooCommerce mobile app', 'woocommerce' );
+	}
+
+	/**
+	 * Content.
+	 *
+	 * @return string
+	 */
+	public function get_content() {
+		return '';
+	}
+
+	/**
+	 * Time.
+	 *
+	 * @return string
+	 */
+	public function get_time() {
+		return '';
+	}
+
+	/**
+	 * Task completion.
+	 *
+	 * @return bool
+	 */
+	public function is_complete() {
+		return get_option( 'woocommerce_admin_dismissed_mobile_app_modal' ) === 'yes';
+	}
+
+	/**
+	 * Task visibility.
+	 *
+	 * @return bool
+	 */
+	public function can_view() {
+		return current_user_can( 'manage_woocommerce' ) && current_user_can( 'install_plugins' );
```

Comment:
LGTM! We're going to add this for **ALL** stores. Is that expected? 🙂

Reply:
I think so, the only B to A/B test it against is .. not having it